tests/workload/basic.robot

```diff
@@ -106,8 +107,7 @@ Create a Hard Link within a Volume
     ...    Should Not Exist         ${DIR2}     AND
     ...    Should Not Exist         ${LINK2}    AND
     ...    Should Not Exist         ${FILE3}
-    ${exdev}=   Evaluate    os.errno.EXDEV    modules=os
-    Link    ${FILE3}  ${LINK2}  code_should_be=${exdev}
+    Link    ${FILE3}  ${LINK2}  code_should_be=${EXDEV}
```

Comment:
Thanks.

Sorry to be picky, but this says "code_should_be" when we are passed a message and not a code.

Why don't we just pass "EXDEV" and let the library lookup the errno for the symbol when it is a string?

Something like:
 
    Link   ${FILE3}   ${LINK2}   code_should_be=EXDEV

Alternatively, you could have a different keyword argument to the Link keyword. Something like;

     Link ...... error_message_should_contain=Invalid cross-device link

But the error message may depend on locale or other things, so feel it would be better to pass the symbol name and let the library resolve that to the errno.

Also, I don't need to create a new variable if the argument is fixed. Before we had ${exdev} as a variable because we looked it by evaluating a python snippet, but you've removed from the test (which is good), but
why do we need ${exdev} now?